arcgis-ios-sdk-samples/Maps/Manage operational layers/MMLLayersViewController.swift

```diff
@@ -15,77 +15,82 @@
 import UIKit
 import ArcGIS
 
-protocol MMLLayersViewControllerDelegate: AnyObject {
-    func layersViewControllerWantsToClose(_ layersViewController: MMLLayersViewController, withDeletedLayers layers: [AGSLayer])
-}
-
 class MMLLayersViewController: UITableViewController {
-    var layers: NSMutableArray!
-    var deletedLayers: [AGSLayer]!
     
-    weak var delegate: MMLLayersViewControllerDelegate?
+    /// Every layer on the map or that could be added to the map.
+    var allLayers: [AGSLayer] = []
+    
+    /// The layers attached to the map.
+    var operationalLayers: NSMutableArray?
```

Comment:
Can you make this not optional?  It would get rid of a lot of forced unwrapping.

Reply:
Made the changes in the latest commit.